packages/circuit-ui/components/ImageInput/ImageInput.tsx

```diff
@@ -45,7 +46,15 @@ export interface ImageInputProps
    * The visual component to render as an image input. It should accept an src
    * prop to render the image.
    */
-  component: ({ src, alt }: { src?: string; alt: string }) => JSX.Element;
+  component: ({
+    src,
+    alt,
+    role,
```

Comment:
I'll take another look at this, see if there's an alternative to avoid it, and if it could be an accessibility regression 🙂